.github/workflows/linux-build.yml

```
@@ -30,7 +30,6 @@ on:
       - "scripts/setup-helper-functions.sh"
       - ".github/workflows/linux-build.yml"
       - ".github/workflows/linux-build-base.yml"
```

```suggestion
      - ".github/workflows/linux-build-base.yml"

```

Just to revert unnecessary changes.